Containerfile

```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM registry.hub.docker.com/library/golang:1.22.2-alpine3.18 as builder
+FROM --platform=$BUILDPLATFORM registry.hub.docker.com/library/golang:1.23.1-alpine3.20 as builder
```

Comment:
It's supposed to happen here: https://github.com/db-operator/db-operator/pull/67

So maybe we can wait a bit?

Reply:
Yes, I will revert, thanks